Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "SpyBug",
     defaultLocalization: "en",
-    platforms: [.iOS(.v15)],
+    platforms: [.iOS(.v16), .visionOS(.v1)],
```

Comment:
Is there a specific reason to drop support for iOS 15?
It's dropping a lot of currently used devices like the old iphone SE and iPhone 8